routes/mapRouters.js

```diff
@@ -0,0 +1,21 @@
+const e = require("express");
+const express = require("express");
+
+const router = express.Router();
+const dev = require("../public/devs.json");
+
+//var name = "Dalia Ramos-Hernandez";
+router.get("/map/get", (req, res) => {
+  res.render("map");
+});
+router.post("/map/get", (req, res) => {
+  var name = req.body.name;
+  const found = dev.find((devObj) => devObj.name === name);
+  if (found) {
+    const singleDev = dev.filter((devObj) => devObj.name === name);
+    res.status(200).json(singleDev);
+  } else {
+    res.status(404).json({ message: "Developer not found" });
+  }
+});
```

Comment:
From what I can tell, the GET /map/get returns the form and POST /map/get checks the file and returns the developer. This is not exactly what we want.